ptm2human.c

```diff
@@ -52,13 +53,14 @@ static const struct option options[] =
     { NULL, 0, 0, 0   },
 };
 
-static const char *optstring = "i:c:Cp0:8:9:2:3:edh";
+static const char *optstring = "i:c:Cp0:8:9:2:3:edhu";
 
 void usage(void)
 {
     printf("Usage: ptm2human [options] -i /TRACE/FILE/PATH\n");
     printf("Options:\n");
-    printf("  -i|--input <trace file>                 Give the trace file\n\n");
+    printf("  -i|--input <trace file>                 Give the trace file\n");
+    printf("  -u|--unaligned                          Trace is unaligned and needs to be aligned by FSYNC\n\n");
```

Comment:
Please use the full name (rather than the abbreviation) in the promt message.

FSYNC -> frame synchronization packet

Reply:
Fixed in 31443f0.